### ISSUE TITLE
Assertion error when dragging windows out of main window

### ISSUE THREAD

**Issue by thaumictom** [codecat/tm-dashboard#66]
The game crashes with an assertion error when you are dragging windows out of the game windows (only works when playing Windowed).

**Comment by codecat** [comment]
I can not reproduce this. You are talking about the widget locator windows, right?

**Comment by thaumictom** [comment]
Indeed, I could only reproduce it with the tachometer. Here is a video showcasing and reproducing said issue:

https://user-images.githubusercontent.com/15818698/221701196-28dfa95c-4282-4623-9c9e-c9ef63b5cf2c.mp4

**Comment by treierxyz** [comment]
I can't reproduce the crash using Proton, but I might have found a related issue. 
When resizing any widget to equal the width or height of the game window, the corresponding position value goes bananas. 

For example, here the game window is 1920x1080. Setting any widget's width to 1920 causes an invalid X position. It's possible this could be causing the crash. Tested w/ Dashboard 1.9.6 on TM2 Canyon. 
![20230917145718_1](https://github.com/codecat/tm-dashboard/assets/33978534/c9124179-2ccc-4dee-8c40-e88fefde9896)

<details>
<summary>Here is the error log:</summary>

```
(14:58:42) 7 ScriptRuntime Dashboard Script exception: Invalid float value (NaN or Inf) passed to y
(14:58:42) 7 ScriptRuntime Dashboard   Dashboard.op/:/Source/Thing.as (line 18, column 3)
(14:58:42) 7 ScriptRuntime Dashboard     #0  void DashboardThing::InternalRender(CSceneVehicleVisState@ vis) (Dashboard.op/:/Source/Thing.as line 19)
(14:58:42) 7 ScriptRuntime Dashboard     #1  void Dashboard::Render() (Dashboard.op/:/Source/Dashboard.as line 54)
(14:58:42) 7 ScriptRuntime Dashboard     #2  void Render() (Dashboard.op/:/Source/Main.as line 23)
```
</details>
